docs/tutorials/vault/creating-vault-issuers.rst

```diff
@@ -55,6 +55,7 @@ We can now create a cluster issuer referencing this secret:
         path: pki_int/sign/example-dot-com
         server: https://vault
         auth:
+          authPath: approle
```

Comment:
This feels a bit awkward - we have `auth.authPath`, which actually only applies to `AppRole` currently, and defaults to something that is specific to the approle auth backend.

I guess this is because token based auth does not use a mounted backend, and we support no others right now.

We can work around this either by making `auth.approle.path` the key to set (and in future, creating `auth.kubernetes.path`, `auth.pki.path` etc). OR we have `auth.path`, which becomes a *required field* *iff* the auth backend selected is *not* 'token'.

I'm keen to choose whatever is most obvious to users 😄

Reply:
`s/authPath/path/g` (as you changed this as part of review 😄)